### PR TITLE
No more directly reading anything from FractalRegsistry

### LINKED ISSUE #2440: Read DAO Name from Subgraph, not directly from FractalRegistry contract
Depends on https://github.com/decentdao/decent-subgraph/issues/18

Once that work is finished and a new subgraph version has been deployed, update this repo to rely on reading that data from the subgraph and not directly from the `FractalRegistry` contract.

### PR DESCRIPTION
**Pull request by adamgall**
Closes #2440 

This PR does two things:

1. Removes unused props (and unused data loaders) from a couple of components.
2. Removes all (only one) direct log reads from FractalRegistry contract.
3. Bumps subgraph version to [v0.1.2](https://github.com/decentdao/decent-subgraph/tree/v0.1.2)

### CODE REVIEW

File: src/hooks/utils/useGetSafeName.ts [hunk 1]
```diff
@@ -31,22 +26,7 @@ export const useGetSafeName = (chainId?: number) => {
...
-      const events = await fractalRegistryContract.getEvents.FractalNameUpdated(
-        { daoAddress: address },
-        { fromBlock: 0n },
-      );
-
-      const latestEvent = events.pop();
-
-      if (latestEvent?.args.daoName) {
-        return latestEvent.args.daoName;
-      } else if (publicClient.chain && demoData[publicClient.chain.id]) {
```
Comment (adamgall) [inline src/hooks/utils/useGetSafeName.ts:49]: As far as I could tell, this code path wasn't ever being hit in practice anyway. Myosin "Organization" page loads just fine without it. Everywhere we are reading DAO names in the app, it's coming from subgraph.

File: src/components/ui/cards/DAONodeInfoCard.tsx [hunk 2]
```diff
@@ -29,7 +27,7 @@ export interface InfoProps extends FlexProps {
  * Simply using the useFractal() hook to get info will end up with the current DAO's
  * context being displayed in ALL the node cards in a hierarchy, which is incorrect.
  */
-export function DAONodeInfoCard({ node, freezeGuard, guardContracts, ...rest }: InfoProps) {
+export function DAONodeInfoCard({ node, ...rest }: InfoProps) {
```
Comment (Da-Colon) [inline src/components/ui/cards/DAONodeInfoCard.tsx:30]: I'm confused why isn't this throwing unused variable warning? weird

**Review by Da-Colon — APPROVED** [review]

File: src/components/ui/cards/DAONodeInfoCard.tsx (hunk 2 above)
Comment (adamgall) [reply src/components/ui/cards/DAONodeInfoCard.tsx:30]: yeah idk

File: src/hooks/utils/useGetSafeName.ts (hunk 1 above)
Comment (adamgall) [reply src/hooks/utils/useGetSafeName.ts:49]: LOL OOPS: https://github.com/decentdao/decent-interface/pull/2487

**Pull request merged**